Code/Framework/AzCore/AzCore/Memory/IAllocator.h

```diff
@@ -100,56 +97,19 @@ namespace AZ
     /**
      * Interface class for all allocators.
      */
-    class IAllocator
+    class IAllocator : public IAllocatorSchema
```

Comment:
Thank you for consolidating `IAllocatorAllocate` interface.
Both `IAllocator` and `IAllocatorSchema` was pretty much the same class

Reply:
yeah, and is being used in mixed scenarios, this is just a quick iteration, hope to get rid of these two interfaces eventually